### PR TITLE
add installation.md

### PR DESCRIPTION
**Pull request by krtk6160**

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Installation
+
+There are multiple ways to install the galoy servers. We recommend first trying it out on your local machine with minikube.
+
+## Local installation
+
+### Prerequisites
+1. Node.js
```
Comment (nicolasburtey) [inline INSTALLATION.md:8]: why not put minikube in prerequisite?

**Review by nicolasburtey — COMMENTED** [review]
just one comment. good otherwise.

**Comment by nicolasburtey** [comment]
we probably want to add some information about running 
1/ the tests locally
2/ graphql locally

File: INSTALLATION.md (hunk 1 above)
Comment (krtk6160) [reply INSTALLATION.md:8]: Fixed

**Comment by krtk6160** [comment]
> we probably want to add some information about running
> 1/ the tests locally
> 2/ graphql locally

Done.

**Pull request merged**